test/playback-watcher.test.js

```diff
@@ -188,7 +239,17 @@ QUnit.test('skips over gap in Chrome due to video underflow', function(assert) {
   assert.equal(vhsVideoUnderflowEvents, 0, 'no video underflow event got triggered');
   assert.equal(hlsVideoUnderflowEvents, 0, 'no video underflow event got triggered');
 
-  this.player.currentTime(13);
+  const mpc = this.player.tech_.vhs.masterPlaylistController_;
+
+  mpc.sourceUpdater_.videoBuffered = () => {
+    return videojs.createTimeRanges([[0, 10], [10, 15]]);
```

Comment:
This (and below): should they be one range? Or meant to have a gap here?

Reply:
I just copied from the other test, but I don't think we need a gap here. removing it.